src/js/HScrollMenu.js

```diff
@@ -16,34 +22,71 @@ export default class HScrollMenu extends React.Component {
         this.hiddenNames.push(name)
     }
 
+    pressResetTimeoutButton(event) { 
+        const count = store.getState().ui[store.getState().activeApp].resetTimeout.resetTimeoutValue/1000;
+
+        this.setState({performInteractionCounter: count});
+        UIController.resetPerformInteractionTimeout('UI');      
+    }
+
+    changeCounter() {
+        this.setState(prevState => ({performInteractionCounter: prevState.performInteractionCounter > 0 ? prevState.performInteractionCounter - 1 : ''}))
+    }
+    componentDidMount() {
+        this.interval = setInterval(() => this.changeCounter(), 1000)
+    }
+    componentWillUnmount() {
+        clearInterval(this.interval)
+    }
+
     render() {
         this.clearHiddenNames();
         let menuItems = this.props.data.map((menuItem) => {
             if (menuItem.hidden) {
                 this.pushHiddenName(menuItem.name)
                 return null;
             }
-            return (<div className="hscrollmenu-block"
+
+            return (
+    
+            <div className="hscrollmenu-block"
                 key={menuItem.cmdID + menuItem.name}>
                     <HScrollMenuItem
                         menuItem={menuItem}
                         isPerformingInteraction={this.props.isPerformingInteraction}
                         interactionId={this.props.interactionId ? this.props.interactionId : 0}
                         theme={this.props.theme}
                         onSelection={this.props.onSelection}/>
-                </div>)
+
+            </div>
+            )
         });
 
         var hiddenCommands = null;
         if (this.hiddenNames.length) {
             hiddenCommands = <MenuFooter textAlign="center"/>
         }
 
-        return (
-            <div className="hscrollmenu">
-                { menuItems }
-                { hiddenCommands }
+        let resetTimeoutPInteraction = undefined;
```

Comment:
@LiliiaShlikhtLuxoft Uninitialized value gets `undefined` by default. Do we need explicitly assign `undefined` ?

Reply:
Fixed